Memento/Views/DetailView.swift

```diff
@@ -77,6 +77,19 @@ struct DetailView: View {
                             }
                         }
                         .buttonStyle(.bordered)
+                        
+                        if let url = item.url {
+                            ShareLink(item: url) {
+                                Label("Share Link", systemImage: "square.and.arrow.up")
+                            }
+                        }
+                        
+                        if let note = item.note, !note.isEmpty {
```

Comment:
change to `else if` so two share buttons dont show up if an item has both a note and a link, share should default to link if both are present.